core/src/com/unciv/logic/automation/unit/UnitAutomation.kt

```diff
@@ -369,8 +369,8 @@ object UnitAutomation {
         val tilesThatCanWalkToAndThenPillage = unitDistanceToTiles
             .filter { it.value.totalDistance < unit.currentMovement }.keys
             .filter { unit.movement.canMoveTo(it) && UnitActionsPillage.canPillage(unit, it)
-                    && (it.canPillageTileImprovement()
-                    || (it.canPillageRoad() && it.getRoadOwner() != null && unit.civ.isAtWarWith(it.getRoadOwner()!!)))}
+                    && ((it.canPillageTileImprovement() && !it.getTileImprovement()!!.hasUnique(UniqueType.DefensiveBonus))
```

Comment:
Units probably should still pillage stat providing forts and/or forts that that damages them